packages/analytics/__tests__/providers/kinesis-firehose/apis/record.test.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+	getEventBuffer,
+	resolveConfig,
+} from '../../../../src/providers/kinesis-firehose/utils';
+import { resolveCredentials } from '../../../../src/utils';
+import {
+	mockConfig,
+	mockCredentialConfig,
+} from '../../../testUtils/mockConstants.test';
+import { record } from '../../../../src/providers/kinesis-firehose';
+import { ConsoleLogger as Logger } from '@aws-amplify/core/lib/Logger';
+import { RecordInput as KinesisFirehoseRecordInput } from '../../../../src/providers/kinesis-firehose/types';
+
+jest.mock('../../../../src/utils');
+jest.mock('../../../../src/providers/kinesis-firehose/utils');
+
+describe('Analytics KinesisFirehose API: record', () => {
+	const mockEvent: KinesisFirehoseRecordInput = {
```

Comment:
Nit: I know it's ultimately an alias but we should probably have these match here for consistency (for future readers)

```suggestion
	const mockInput: KinesisFirehoseRecordInput = {
```

Ditto for Kinesis test

Reply:
👍 will update the naming.